### PR TITLE
docs: Categorize docs files by services

### PR DESCRIPTION
**Pull request by haein-side**
## Summary
- Categorized docs files by service

## Changes
- Categorized resources by service with reference to [official documents](https://developer.hashicorp.com/terraform/registry/providers/docs#subcategories)
- Added `subcategory` field to top of markdown document

```
---
subcategory: "Service Name"
---
```
- Named service name with reference to [ncloud api docs](https://guide.ncloud-docs.com/docs/ko/home) and [ncloud console](https://www.ncloud.com/)
ex. `Cloud Data Streaming Service`, `NAS Volume`, `Kubernetes Service`, `Search Engine Service`, `Login Key`
- Grouped `regions.md` and `zones.md` in `Meta Data Sources`

### CODE REVIEW

**Review by wonchulee — APPROVED** [review]
LGTM 👍

File: docs/data-sources/launch_configuration.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+subcategory: "Launch Configuration"
```
Comment (youngmn) [inline docs/data-sources/launch_configuration.md]: 최근에 launch configuration 은 Auto Scaling 카테고리로 분류되어 통합되었습니다.

https://github.com/NaverCloudPlatform/terraform-provider-ncloud/pull/318

File: docs/resources/launch_configuration.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+subcategory: "Launch Configuration"
```
Comment (youngmn) [inline docs/resources/launch_configuration.md]: 최근에 launch configuration 은 Auto Scaling 카테고리로 분류되어 통합되었습니다.

https://github.com/NaverCloudPlatform/terraform-provider-ncloud/pull/318

File: docs/data-sources/launch_configuration.md (hunk 1 above)
Comment (haein-side) [reply docs/data-sources/launch_configuration.md]: 수정 완료했습니다!! 피드백 감사합니다!

File: docs/resources/launch_configuration.md (hunk 2 above)
Comment (haein-side) [reply docs/resources/launch_configuration.md]: 수정 완료했습니다!! 피드백 감사합니다!

**Review by youngmn — APPROVED** [review]
LGTM 👍

**Pull request merged**